tools/docker/Dockerfile

```diff
@@ -24,17 +26,29 @@ RUN apt-get -qqy update \
     locales \
     openjdk-8-jre-headless \
     pulseaudio \
-    python \
-    python-pip \
+    python2 \
+    python2-dev \
     python3 \
+    python3-dev \
     python3-pip \
+    software-properties-common \
     qemu-kvm \
     tzdata \
     sudo \
     unzip \
     wget \
     xvfb
 
+# python3.6 is not available by default in new versions of Ubuntu.
+RUN apt-add-repository -y ppa:deadsnakes/ppa
+
+# Ensure that scripts referencing `/usr/bin/env python` work.
+RUN update-alternatives --install /bin/python python /usr/bin/python2 1
```

Comment:
Nit: let's install the recommended `python-is-python2` package to take care of this: https://wiki.ubuntu.com/FocalFossa/ReleaseNotes#Python3_by_default